sale_force_whole_invoiceability/views/sale_order_views.xml

```diff
@@ -5,14 +5,14 @@
     <record id="view_order_form" model="ir.ui.view">
         <field name="model">sale.order</field>
         <field name="inherit_id" ref="sale.view_order_form" />
-        <field name="groups_id" eval="[(4,ref('sales_team.group_sale_manager'))]" />
```

Comment:
Why has it been removed?

Reply:
Because must be in a field declaration and not in the view.